controllers/llamastackdistribution_controller.go

```diff
@@ -225,6 +225,11 @@ func (r *LlamaStackDistributionReconciler) reconcileResources(ctx context.Contex
 		}
 	}
 
+	// Reconcile manifest-based resources
+	if err := r.reconcileManifestResources(ctx, instance); err != nil {
```

Comment:
This is a duplicate of https://github.com/llamastack/llama-stack-k8s-operator/pull/105/files#diff-5dcdc1c5c5637dad847c56c4f2b187f49894b10769974fcef0bc3a8d303dc312R239

Reply:
removed!